globals.json

```diff
@@ -745,5 +748,16 @@
 		"Path": false,
 		"Progress": false,
 		"Ref": false
+	},
+	"serviceworker": {
+		"Cache": false,
+		"CacheStorage": false,
+		"Client": false,
+		"Clients": false,
+		"ExtendableEvent": false,
```

Comment:
Yes, `ExtendableMessageEvent` is missing because I didn't find it while testing manually in Chrome. Should we include all globals in the spec right away?